google/cloud/storage/README.md

```diff
@@ -44,7 +44,14 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
 
 ## Release Notes
 
-### v1.8.x - TBD
+### v1.9.x - TBD
+
+### v1.8.x - 2020-01
+
+* feat: implement non-resumable parallel uploads (#3279)
+* feat: support optionsRequestedPolicyVersion query parameter in GetBucketIamPolicy
+* feat: move creating prefix marker to ComposeMany (#3296)
```

Comment:
This is actually a breaking change, but the merged message does not say so, see #3296 for details.

Reply:
Done.